app/Http/Traits/IndexElastic.php

```diff
@@ -728,6 +729,7 @@ public function indexElasticTools(int $toolId, bool $returnParams = false): null
                 'tags' => $tags,
                 'datasetTitles' => $datasetTitles,
                 'dataProviderColl' => $dataProviderColl,
+                'dataProvider' => $tool['team']['name'],
```

Comment:
This might fall over if `team` is `null`.  Maybe worth updating to:
```suggestion
                'dataProvider' => $tool['team']['name'] ?? null,
```